mypy/waiter.py

```diff
@@ -161,12 +189,14 @@ def _record_time(self, name: str, elapsed_time: float) -> None:
 
     def _poll_current(self) -> Tuple[int, int]:
         while True:
-            time.sleep(.05)
+            time.sleep(.01)
```

Comment:
@ambv Actually, I'm only *hoping* so. On the positive side, with ~140 tasks, we're cutting down on the average delay of 0.05 per task, or ~7 sec for the current test suite (well, for slower it's actually half of that, since on average they finishe in the middle of the wait interval). Spread across (say) 4 workers, it only reduces total wall time by a quarter of that, or ~2 sec.

But OTOH, reducing the wait time also increases the number of context switches to the main thread and the CPU load of the main thread (i.e., the thread that is waiting for the next task to finish); both would cause slowdown in the worker threads. I hope it's not enough to become noticeable at 10 ms wait time, given that a context switch is just a few microseconds, and the runtime of the loop seems to be pretty short too. But I'm too lazy to test it.